.vscode/tasks.json

```diff
@@ -114,13 +117,45 @@
       "problemMatcher": "$tsc"
     },
     {
-      "taskName": "Clean all generated files",
-      "isShellCommand": true,
+      "label": "Clean all generated files",
```

Comment:
Can we call this `Clean project`? This is a suggestion not a blocker. (I prefer simple names).